### ISSUE TITLE
Build fails in new v0.9.0 s2i-thoth-ubi8-py36 base image

### ISSUE THREAD

**Issue by 4n4nd** [thoth-station/s2i-thoth#39]
**Describe the bug**
Build fails due to a missing dependency which can be seen as installed in the build logs.
This fails is in the newer version v0.9.0 but this build works in v0.8.0 of the image  thoth-station/s2i-thoth-ubi8-py36.

**Environment variables used for s2i configuration**
The build config can be found here: https://github.com/AICoE/prometheus-anomaly-detector/blob/master/openshift/oc-image-build-template.yaml

**Expected behavior**
Build finishes and image is created on openshift

**Actual behavior**
Build Assembly fails on openshift

**Build log**
<details><summary>Failed Build Log:</summary>
<p>

```
Thoth stack analysis failed with the following log:
2020-02-25 16:38:35,121 [141] DEBUG    thamos: Debug mode turned on
2020-02-25 16:38:35,121 [141] DEBUG    thamos: Thamos version: '0.8.1'
2020-02-25 16:38:35,122 [141] INFO     thamos.config: Expanding configuration file based on environment variables
2020-02-25 16:38:35,164 [141] WARNING  thamos.config: TLS verification turned off, its highly recommended to use a secured connection, see configuration file for configuration options
2020-02-25 16:38:35,164 [141] DEBUG    thamos.lib: Using API: https://khemenu.thoth-station.ninja/api/v1 
2020-02-25 16:38:35,498 [141] CRITICAL root: Traceback (most recent call last):
  File "/opt/app-root/bin/thamos", line 8, in <module>
    sys.exit(cli())
  File "/opt/app-root/lib/python3.6/site-packages/click/core.py", line 764, in __call__
    return self.main(*args, **kwargs)
  File "/opt/app-root/lib/python3.6/site-packages/click/core.py", line 717, in main
    rv = self.invoke(ctx)
  File "/opt/app-root/lib/python3.6/site-packages/click/core.py", line 1137, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/opt/app-root/lib/python3.6/site-packages/click/core.py", line 956, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/opt/app-root/lib/python3.6/site-packages/click/core.py", line 555, in invoke
    return callback(*args, **kwargs)
  File "/opt/app-root/lib/python3.6/site-packages/thamos/cli.py", line 488, in log
    log_str = get_log()
  File "/opt/app-root/lib/python3.6/site-packages/thamos/lib.py", line 81, in wrapper
    result = func(api_client, *args, **kwargs)
  File "/opt/app-root/lib/python3.6/site-packages/thamos/lib.py", line 583, in get_log
    return method(analysis_id).log
  File "/opt/app-root/lib/python3.6/site-packages/thamos/swagger_client/thoth/advise_api.py", line 146, in get_advise_python_log
    (data) = self.get_advise_python_log_with_http_info(analysis_id, **kwargs)  # noqa: E501
  File "/opt/app-root/lib/python3.6/site-packages/thamos/swagger_client/thoth/advise_api.py", line 219, in get_advise_python_log_with_http_info
    collection_formats=collection_formats)
  File "/opt/app-root/lib/python3.6/site-packages/thamos/swagger_client/api_client.py", line 321, in call_api
    _preload_content, _request_timeout)
  File "/opt/app-root/lib/python3.6/site-packages/thamos/swagger_client/api_client.py", line 152, in __call_api
    _request_timeout=_request_timeout)
  File "/opt/app-root/lib/python3.6/site-packages/thamos/swagger_client/api_client.py", line 342, in request
    headers=headers)
  File "/opt/app-root/lib/python3.6/site-packages/thamos/swagger_client/rest.py", line 237, in GET
    query_params=query_params)
  File "/opt/app-root/lib/python3.6/site-packages/thamos/swagger_client/rest.py", line 227, in request
    raise ApiException(http_resp=r)
thamos.swagger_client.rest.ApiException: (404)
Reason: NOT FOUND
HTTP response headers: HTTPHeaderDict({'Server': 'gunicorn/20.0.4', 'Date': 'Tue, 25 Feb 2020 16:38:35 GMT', 'Content-Type': 'application/json', 'Content-Length': '143', 'X-Thoth-Version': '0.6.0-dev', 'Access-Control-Allow-Origin': '*', 'Set-Cookie': 'ae5b4faaab1fe6375d62dbc3b1efaf0d=7c1de92fafc113faef6888d79cbe7a0b; path=/; HttpOnly; Secure', 'Cache-control': 'private'})
HTTP response body: {
  "error": "No analysis with id adviser-e9acbf03-e6ed7cd8 was found",
  "parameters": {
    "analysis_id": "adviser-e9acbf03-e6ed7cd8"
  }
}
Restoring previous Pipfile.lock
Thoth advises are not activated
~
---> Installing application source ...
---> Installing dependencies via micropipenv ...
Collecting alembic==1.4.0
Downloading alembic-1.4.0.tar.gz (1.1 MB)
Requirement already satisfied: certifi==2019.11.28 in /opt/app-root/lib/python3.6/site-packages (from -r /tmp/requirements.txt15haqsxm (line 7)) (2019.11.28)
Requirement already satisfied: chardet==3.0.4 in /opt/app-root/lib/python3.6/site-packages (from -r /tmp/requirements.txt15haqsxm (line 10)) (3.0.4)
Requirement already satisfied: click==7.0 in /opt/app-root/lib/python3.6/site-packages (from -r /tmp/requirements.txt15haqsxm (line 13)) (7.0)
Collecting cloudpickle==1.3.0
Downloading cloudpickle-1.3.0-py2.py3-none-any.whl (26 kB)
Collecting configparser==4.0.2
Downloading configparser-4.0.2-py2.py3-none-any.whl (22 kB)
Collecting convertdate==2.2.0
Downloading convertdate-2.2.0-py3-none-any.whl (37 kB)
Collecting cycler==0.10.0
Downloading cycler-0.10.0-py2.py3-none-any.whl (6.5 kB)
Collecting cython==0.29.15
Downloading Cython-0.29.15-cp36-cp36m-manylinux1_x86_64.whl (2.1 MB)
Collecting databricks-cli==0.9.1
Downloading databricks-cli-0.9.1.tar.gz (45 kB)
Collecting dateparser==0.7.2
Downloading dateparser-0.7.2-py2.py3-none-any.whl (352 kB)
Collecting docker==4.2.0
Downloading docker-4.2.0-py2.py3-none-any.whl (143 kB)
Collecting entrypoints==0.3
Downloading entrypoints-0.3-py2.py3-none-any.whl (11 kB)
Collecting fbprophet==0.5
Downloading fbprophet-0.5.tar.gz (49 kB)
Collecting flask==1.1.1
Downloading Flask-1.1.1-py2.py3-none-any.whl (94 kB)
Collecting gitdb2==2.0.6
Downloading gitdb2-2.0.6-py2.py3-none-any.whl (63 kB)
Collecting gitpython==3.0.7
Downloading GitPython-3.0.7-py3-none-any.whl (451 kB)
Collecting gorilla==0.3.0
Downloading gorilla-0.3.0-py2.py3-none-any.whl (11 kB)
Collecting gunicorn==20.0.4
Downloading gunicorn-20.0.4-py2.py3-none-any.whl (77 kB)
Collecting holidays==0.9.12
Downloading holidays-0.9.12.tar.gz (85 kB)
Collecting idna==2.8
Downloading idna-2.8-py2.py3-none-any.whl (58 kB)
Collecting itsdangerous==1.1.0
Downloading itsdangerous-1.1.0-py2.py3-none-any.whl (16 kB)
Requirement already satisfied: jinja2==2.11.1 in /opt/app-root/lib/python3.6/site-packages (from -r /tmp/requirements.txt15haqsxm (line 98)) (2.11.1)
Collecting kiwisolver==1.1.0
Downloading kiwisolver-1.1.0-cp36-cp36m-manylinux1_x86_64.whl (90 kB)
Collecting lunardate==0.2.0
Downloading lunardate-0.2.0-py3-none-any.whl (5.6 kB)
Collecting mako==1.1.1
Downloading Mako-1.1.1.tar.gz (468 kB)
Requirement already satisfied: markupsafe==1.1.1 in /opt/app-root/lib/python3.6/site-packages (from -r /tmp/requirements.txt15haqsxm (line 145)) (1.1.1)
Collecting matplotlib==3.1.3
Downloading matplotlib-3.1.3-cp36-cp36m-manylinux1_x86_64.whl (13.1 MB)
Collecting mlflow==1.6.0
Downloading mlflow-1.6.0.tar.gz (15.9 MB)
Collecting numpy==1.18.1
Downloading numpy-1.18.1-cp36-cp36m-manylinux1_x86_64.whl (20.1 MB)
Collecting pandas==1.0.1
Downloading pandas-1.0.1-cp36-cp36m-manylinux1_x86_64.whl (10.1 MB)
Collecting prometheus-api-client==0.2.0
Downloading prometheus_api_client-0.2.0-py3-none-any.whl (12 kB)
Collecting prometheus-client==0.7.1
Downloading prometheus_client-0.7.1.tar.gz (38 kB)
Collecting prometheus-flask-exporter==0.12.2
Downloading prometheus_flask_exporter-0.12.2.tar.gz (18 kB)
Collecting protobuf==3.11.3
Downloading protobuf-3.11.3-cp36-cp36m-manylinux1_x86_64.whl (1.3 MB)
Collecting pymeeus==0.3.6
Downloading PyMeeus-0.3.6.tar.gz (736 kB)
Requirement already satisfied: pyparsing==2.4.6 in /opt/app-root/lib/python3.6/site-packages (from -r /tmp/requirements.txt15haqsxm (line 262)) (2.4.6)
Collecting pystan==2.19.1.1
Downloading pystan-2.19.1.1-cp36-cp36m-manylinux1_x86_64.whl (67.3 MB)
Requirement already satisfied: python-dateutil==2.8.1 in /opt/app-root/lib/python3.6/site-packages (from -r /tmp/requirements.txt15haqsxm (line 282)) (2.8.1)
Collecting python-editor==1.0.4
Downloading python_editor-1.0.4-py3-none-any.whl (4.9 kB)
Requirement already satisfied: pytz==2019.3 in /opt/app-root/lib/python3.6/site-packages (from -r /tmp/requirements.txt15haqsxm (line 289)) (2019.3)
Requirement already satisfied: pyyaml==5.3 in /opt/app-root/lib/python3.6/site-packages (from -r /tmp/requirements.txt15haqsxm (line 292)) (5.3)
Collecting querystring-parser==1.2.4
Downloading querystring_parser-1.2.4.tar.gz (5.5 kB)
Collecting regex==2020.1.8
Downloading regex-2020.1.8-cp36-cp36m-manylinux2010_x86_64.whl (689 kB)
Collecting requests==2.22.0
Downloading requests-2.22.0-py2.py3-none-any.whl (57 kB)
Collecting retrying==1.3.3
Downloading retrying-1.3.3.tar.gz (10 kB)
Collecting schedule==0.6.0
Downloading schedule-0.6.0-py2.py3-none-any.whl (8.7 kB)
Collecting setuptools-git==1.2
Downloading setuptools_git-1.2-py2.py3-none-any.whl (10 kB)
Collecting simplejson==3.17.0
Downloading simplejson-3.17.0.tar.gz (83 kB)
Requirement already satisfied: six==1.14.0 in /opt/app-root/lib/python3.6/site-packages (from -r /tmp/requirements.txt15haqsxm (line 356)) (1.14.0)
Collecting smmap2==2.0.5
Downloading smmap2-2.0.5-py2.py3-none-any.whl (24 kB)
Collecting sqlalchemy==1.3.13
Downloading SQLAlchemy-1.3.13.tar.gz (6.0 MB)
Collecting sqlparse==0.3.0
Downloading sqlparse-0.3.0-py2.py3-none-any.whl (39 kB)
Collecting tabulate==0.8.6
Downloading tabulate-0.8.6.tar.gz (45 kB)
Collecting tornado==6.0.3
Downloading tornado-6.0.3.tar.gz (482 kB)
Requirement already satisfied: tzlocal==2.0.0 in /opt/app-root/lib/python3.6/site-packages (from -r /tmp/requirements.txt15haqsxm (line 377)) (2.0.0)
Requirement already satisfied: urllib3==1.25.8 in /opt/app-root/lib/python3.6/site-packages (from -r /tmp/requirements.txt15haqsxm (line 380)) (1.25.8)
Collecting waitress==1.4.3
Downloading waitress-1.4.3-py2.py3-none-any.whl (148 kB)
Requirement already satisfied: websocket-client==0.57.0 in /opt/app-root/lib/python3.6/site-packages (from -r /tmp/requirements.txt15haqsxm (line 386)) (0.57.0)
Collecting werkzeug==1.0.0
Downloading Werkzeug-1.0.0-py2.py3-none-any.whl (298 kB)
Building wheels for collected packages: alembic, databricks-cli, fbprophet, holidays, mako, mlflow, prometheus-client, prometheus-flask-exporter, pymeeus, querystring-parser, retrying, simplejson, sqlalchemy, tabulate, tornado
Building wheel for alembic (setup.py): started
Building wheel for alembic (setup.py): finished with status 'done'
Created wheel for alembic: filename=alembic-1.4.0-py2.py3-none-any.whl size=156760 sha256=b01ccf6870777997742b369b2391eeb9d841b72b4834b5b9b9cef914cb6fa245
Stored in directory: /tmp/pip-ephem-wheel-cache-_ksyiykr/wheels/54/88/28/d771a55dfb3c62af8b3358c60f8034edcb5c9a57d44a9024cf
Building wheel for databricks-cli (setup.py): started
Building wheel for databricks-cli (setup.py): finished with status 'done'
Created wheel for databricks-cli: filename=databricks_cli-0.9.1-py3-none-any.whl size=83762 sha256=8bc574021f96d77ec99ffb819726ec7464098fa03ed5c13bdbf2dd8d2c102ee3
Stored in directory: /tmp/pip-ephem-wheel-cache-_ksyiykr/wheels/3d/27/8f/ccd6bc76a2efcabd1ed7f8656ac432f7fb165d62f2c7df9cf8
Building wheel for fbprophet (setup.py): started
Building wheel for fbprophet (setup.py): finished with status 'error'
Running setup.py clean for fbprophet
  ERROR: Command errored out with exit status 1:
   command: /opt/app-root/bin/python3.6 -u -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-install-kyn0u9d4/fbprophet/setup.py'"'"'; __file__='"'"'/tmp/pip-install-kyn0u9d4/fbprophet/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' bdist_wheel -d /tmp/pip-wheel-egcawq_f
       cwd: /tmp/pip-install-kyn0u9d4/fbprophet/
  Complete output (36 lines):
  running bdist_wheel
  running build
  running build_py
  creating build
  creating build/lib
  creating build/lib/fbprophet
  creating build/lib/fbprophet/stan_model
  Traceback (most recent call last):
    File "<string>", line 1, in <module>
    File "/tmp/pip-install-kyn0u9d4/fbprophet/setup.py", line 120, in <module>
      """
    File "/opt/app-root/lib/python3.6/site-packages/setuptools/__init__.py", line 129, in setup
      return distutils.core.setup(**attrs)
    File "/usr/lib64/python3.6/distutils/core.py", line 148, in setup
      dist.run_commands()
    File "/usr/lib64/python3.6/distutils/dist.py", line 955, in run_commands
      self.run_command(cmd)
    File "/usr/lib64/python3.6/distutils/dist.py", line 974, in run_command
      cmd_obj.run()
    File "/opt/app-root/lib/python3.6/site-packages/wheel/bdist_wheel.py", line 201, in run
      self.run_command('build')
    File "/usr/lib64/python3.6/distutils/cmd.py", line 313, in run_command
      self.distribution.run_command(command)
    File "/usr/lib64/python3.6/distutils/dist.py", line 974, in run_command
      cmd_obj.run()
    File "/usr/lib64/python3.6/distutils/command/build.py", line 135, in run
      self.run_command(cmd_name)
    File "/usr/lib64/python3.6/distutils/cmd.py", line 313, in run_command
      self.distribution.run_command(command)
    File "/usr/lib64/python3.6/distutils/dist.py", line 974, in run_command
      cmd_obj.run()
    File "/tmp/pip-install-kyn0u9d4/fbprophet/setup.py", line 44, in run
      build_stan_model(target_dir)
    File "/tmp/pip-install-kyn0u9d4/fbprophet/setup.py", line 27, in build_stan_model
      from pystan import StanModel
  ModuleNotFoundError: No module named 'pystan'
  ----------------------------------------
  ERROR: Failed building wheel for fbprophet
Building wheel for holidays (setup.py): started
Building wheel for holidays (setup.py): finished with status 'done'
Created wheel for holidays: filename=holidays-0.9.12-py3-none-any.whl size=46763 sha256=505424db51ebfd91d2e62a1d461b10f073a59e396eca8e2e5a5ff25da4f689c6
Stored in directory: /tmp/pip-ephem-wheel-cache-_ksyiykr/wheels/02/33/ad/d2536fe7f536284398d86acc7bd9fb3d2a9eabbd66050c2e2b
Building wheel for mako (setup.py): started
Building wheel for mako (setup.py): finished with status 'done'
Created wheel for mako: filename=Mako-1.1.1-py3-none-any.whl size=74217 sha256=9606b40c223060650af9e155f31d57189eec06b88cbd33c526f18bc604c7ab7e
Stored in directory: /tmp/pip-ephem-wheel-cache-_ksyiykr/wheels/43/b1/7c/f14ef20f4683e5087ae684c6447194e09695315f20b9c45575
Building wheel for mlflow (setup.py): started
Building wheel for mlflow (setup.py): finished with status 'done'
Created wheel for mlflow: filename=mlflow-1.6.0-py3-none-any.whl size=16098516 sha256=8f119cfb3365facdc4a63030ca6e9bcd247eccc6d45aa979f4fa4325ca8d9a88
Stored in directory: /tmp/pip-ephem-wheel-cache-_ksyiykr/wheels/52/17/45/1370dd1b969ca154f4cd7db298a76d17b2a817426c8c9bf1e5
Building wheel for prometheus-client (setup.py): started
Building wheel for prometheus-client (setup.py): finished with status 'done'
Created wheel for prometheus-client: filename=prometheus_client-0.7.1-py3-none-any.whl size=41411 sha256=82432776354e7a4d8a026905ae749220b4677a6d2c4a7af4af3c256c3458da95
Stored in directory: /tmp/pip-ephem-wheel-cache-_ksyiykr/wheels/1d/4a/79/a3ad3f74b3495b4555359375ca33ad7b64e77f8b7a53c8894f
Building wheel for prometheus-flask-exporter (setup.py): started
Building wheel for prometheus-flask-exporter (setup.py): finished with status 'done'
Created wheel for prometheus-flask-exporter: filename=prometheus_flask_exporter-0.12.2-py3-none-any.whl size=14036 sha256=d385d952067956e40b6fbc5ded00e673751ed5585a149afd41b149892529c0df
Stored in directory: /tmp/pip-ephem-wheel-cache-_ksyiykr/wheels/45/6e/cc/3691602eeca883fa3db046605c675989117c9dbfa13ed46f34
Building wheel for pymeeus (setup.py): started
Building wheel for pymeeus (setup.py): finished with status 'done'
Created wheel for pymeeus: filename=PyMeeus-0.3.6-py3-none-any.whl size=687691 sha256=310d3e5028c1fe7bfe8588b4ab8acedb5a2b9c09946a0bf00948a3236cc72c8a
Stored in directory: /tmp/pip-ephem-wheel-cache-_ksyiykr/wheels/5d/55/8f/0f14cf1d08b90255db77f770f65e183e6494c90570181a061e
Building wheel for querystring-parser (setup.py): started
Building wheel for querystring-parser (setup.py): finished with status 'done'
Created wheel for querystring-parser: filename=querystring_parser-1.2.4-py3-none-any.whl size=7083 sha256=d7b4fef4f9cb1867a492da546a2604aeefec8469c995f161b5375b124abc90e2
Stored in directory: /tmp/pip-ephem-wheel-cache-_ksyiykr/wheels/87/da/0d/a9e8051c7c19128c271107462cd444b37afc22aaed15061ed5
Building wheel for retrying (setup.py): started
Building wheel for retrying (setup.py): finished with status 'done'
Created wheel for retrying: filename=retrying-1.3.3-py3-none-any.whl size=6236 sha256=fcaa04480b723c7bcfddfe10abcca84fb92443cb6e213a68506c5e092401fe6d
Stored in directory: /tmp/pip-ephem-wheel-cache-_ksyiykr/wheels/ac/cb/8a/b27bf6323e2f4c462dcbf77d70b7c5e7868a7fbe12871770cf
Building wheel for simplejson (setup.py): started
Building wheel for simplejson (setup.py): finished with status 'done'
Created wheel for simplejson: filename=simplejson-3.17.0-cp36-cp36m-linux_x86_64.whl size=118459 sha256=70b79cb1ba994065ba9597d92f7b38f0f47e92ccdda2e9bb98b9709dcdd42671
Stored in directory: /tmp/pip-ephem-wheel-cache-_ksyiykr/wheels/c2/95/a2/477b46cfe980061fddc4cb76ed4657eceec35d2b2090355e41
Building wheel for sqlalchemy (setup.py): started
Building wheel for sqlalchemy (setup.py): finished with status 'done'
Created wheel for sqlalchemy: filename=SQLAlchemy-1.3.13-cp36-cp36m-linux_x86_64.whl size=1222132 sha256=6a2568d0f5e2217199587550a98543c107f2f50f5e254697d2ee3bd3a148ccea
Stored in directory: /tmp/pip-ephem-wheel-cache-_ksyiykr/wheels/28/3e/f9/8eca04781258bb6956ffba37e4e6e6951e5b3a16d4494b91cb
Building wheel for tabulate (setup.py): started
Building wheel for tabulate (setup.py): finished with status 'done'
Created wheel for tabulate: filename=tabulate-0.8.6-py3-none-any.whl size=22472 sha256=13fa492e70f5f7e4466a11a00b8be51b278316d537e49f6daa6c21d79fb57ffc
Stored in directory: /tmp/pip-ephem-wheel-cache-_ksyiykr/wheels/f1/f8/e3/a65c54a883edf04f7d6b0eda5486e6f27279354b84fd932da2
Building wheel for tornado (setup.py): started
Building wheel for tornado (setup.py): finished with status 'done'
Created wheel for tornado: filename=tornado-6.0.3-cp36-cp36m-linux_x86_64.whl size=423813 sha256=a7bf691abb5f238821d92eadf02796a240c7e563d51cb9edd073734c99e07924
Stored in directory: /tmp/pip-ephem-wheel-cache-_ksyiykr/wheels/b2/92/5c/3bfc125cdf46ab0487727e574f513f9568e9b7974b15237abf
Successfully built alembic databricks-cli holidays mako mlflow prometheus-client prometheus-flask-exporter pymeeus querystring-parser retrying simplejson sqlalchemy tabulate tornado
Failed to build fbprophet
Installing collected packages: alembic, cloudpickle, configparser, convertdate, cycler, cython, databricks-cli, dateparser, docker, entrypoints, fbprophet, flask, gitdb2, gitpython, gorilla, gunicorn, holidays, idna, itsdangerous, kiwisolver, lunardate, mako, matplotlib, mlflow, numpy, pandas, prometheus-api-client, prometheus-client, prometheus-flask-exporter, protobuf, pymeeus, pystan, python-editor, querystring-parser, regex, requests, retrying, schedule, setuptools-git, simplejson, smmap2, sqlalchemy, sqlparse, tabulate, tornado, waitress, werkzeug
Running setup.py install for fbprophet: started
Running setup.py install for fbprophet: finished with status 'error'
    ERROR: Command errored out with exit status 1:
     command: /opt/app-root/bin/python3.6 -u -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-install-kyn0u9d4/fbprophet/setup.py'"'"'; __file__='"'"'/tmp/pip-install-kyn0u9d4/fbprophet/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' install --record /tmp/pip-record-ynd_venk/install-record.txt --single-version-externally-managed --compile --install-headers /opt/app-root/include/site/python3.6/fbprophet
         cwd: /tmp/pip-install-kyn0u9d4/fbprophet/
    Complete output (38 lines):
    running install
    running build
    running build_py
    creating build
    creating build/lib
    creating build/lib/fbprophet
    creating build/lib/fbprophet/stan_model
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-install-kyn0u9d4/fbprophet/setup.py", line 120, in <module>
        """
      File "/opt/app-root/lib/python3.6/site-packages/setuptools/__init__.py", line 129, in setup
        return distutils.core.setup(**attrs)
      File "/usr/lib64/python3.6/distutils/core.py", line 148, in setup
        dist.run_commands()
      File "/usr/lib64/python3.6/distutils/dist.py", line 955, in run_commands
        self.run_command(cmd)
      File "/usr/lib64/python3.6/distutils/dist.py", line 974, in run_command
        cmd_obj.run()
      File "/opt/app-root/lib/python3.6/site-packages/setuptools/command/install.py", line 61, in run
        return orig.install.run(self)
      File "/usr/lib64/python3.6/distutils/command/install.py", line 556, in run
        self.run_command('build')
      File "/usr/lib64/python3.6/distutils/cmd.py", line 313, in run_command
        self.distribution.run_command(command)
      File "/usr/lib64/python3.6/distutils/dist.py", line 974, in run_command
        cmd_obj.run()
      File "/usr/lib64/python3.6/distutils/command/build.py", line 135, in run
        self.run_command(cmd_name)
      File "/usr/lib64/python3.6/distutils/cmd.py", line 313, in run_command
        self.distribution.run_command(command)
      File "/usr/lib64/python3.6/distutils/dist.py", line 974, in run_command
        cmd_obj.run()
      File "/tmp/pip-install-kyn0u9d4/fbprophet/setup.py", line 44, in run
        build_stan_model(target_dir)
      File "/tmp/pip-install-kyn0u9d4/fbprophet/setup.py", line 27, in build_stan_model
        from pystan import StanModel
    ModuleNotFoundError: No module named 'pystan'
    ----------------------------------------
ERROR: Command errored out with exit status 1: /opt/app-root/bin/python3.6 -u -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-install-kyn0u9d4/fbprophet/setup.py'"'"'; __file__='"'"'/tmp/pip-install-kyn0u9d4/fbprophet/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' install --record /tmp/pip-record-ynd_venk/install-record.txt --single-version-externally-managed --compile --install-headers /opt/app-root/include/site/python3.6/fbprophet Check the logs for full command output.
error: build error: non-zero (13) exit code from quay.io/thoth-station/s2i-thoth-ubi8-py36:latest

```
</p>
</details>

**Comment by fridex** [comment]
thanks for the report @4n4nd 

This looks like an issue with the relative ordering of dependencies installed (pipenv tries to install failed dependencies again).

Thinking about the fix here... we can try to either reorder dependencies or create similar logic as pipenv has (which would make micropipenv slightly more complicated piece of software).

**Comment by fridex** [comment]
@4n4nd could you please verify the fix works for you? The newly released images in version v0.10.0 should address the issue. Thanks!

**Comment by 4n4nd** [comment]
It works! I tested the new image v10.0.
Thanks!
